src/virt.c

```diff
@@ -1143,6 +1140,18 @@ static int refresh_lists(struct lv_read_instance *inst) {
         goto cont;
       }
 
+      status = virDomainGetInfo(dom, &info);
+      if (status != 0) {
+        ERROR(PLUGIN_NAME " plugin: virDomainGetInfo failed with status %i.",
```

Comment:
There are two calls directly above, virDomainLookupByID() and virDomainGetName(), so this exception must be rather rare situation.